iptables/iptables.go

```diff
@@ -49,8 +49,8 @@ func (e *Error) Error() string {
 // IsNotExist returns true if the error is due to the chain or rule not existing
 func (e *Error) IsNotExist() bool {
 	return e.ExitStatus() == 1 &&
-		strings.Contains(e.msg, fmt.Sprintf("%s: Bad rule (does a matching rule exist in that chain?).\n", getIptablesCommand(e.proto))) ||
-		strings.Contains(e.msg, fmt.Sprintf("%s: No chain/target/match by that name.\n", getIptablesCommand(e.proto)))
+		(strings.Contains(e.msg, fmt.Sprintf("%s: Bad rule (does a matching rule exist in that chain?).\n", getIptablesCommand(e.proto))) ||
```

Comment:
What about un-nesting these statements a bit? The `!= 1` can be manually short-circuited to `true`, the two get-proto calls can be unified, and the `Sprintf` results can be assigned to their own variables.

That should result in a slightly more version function, but an easier to read `return Contains(foo) || Contains(bar)` statement.

Reply:
done, 
I think that `ExistStatus != 1` should return `false`, right?

Reply:
Indeed, sorry for the confusion.